web/src/pages/Rsvp.test.jsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+
+import Rsvp from 'huyan/pages/Rsvp';
+
+test('renders RSVP page', () => {
+   const text = 'RSVP';
+   render(<Rsvp />);
+   expect(screen.getByText(text)).toBeInTheDocument();
```

Comment:
I think we can inline `text` since it isn't used elsewhere?

```suggestion
   render(<Rsvp />);
   expect(screen.getByText('RSVP')).toBeInTheDocument();
```